lib/senders/email.js

```diff
@@ -88,6 +89,21 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0a84ff; text-decoration: none; font-family: sans-serif;"'
   }
 
+  /**
+   * Make given string safer for emails
+   * @param str
+   * @returns String
+   */
+  function makeSubjectSafer(str) {
```

Comment:
@rfk did you have something like this in mind for the final `subject` field? Additional validation?

Reply:
Hm instead of doing this subject validation I'm wondering if the validator from the oauth server is enough here `Joi.string().max(256)`